packages/firestore/src/core/bundle.ts

```diff
@@ -180,9 +180,11 @@ export class BundleLoader {
     this.progress.bytesLoaded += element.byteLength;
 
     let documentsLoaded = this.progress.documentsLoaded;
+    let loadedNamedQuery = false;
 
     if (element.payload.namedQuery) {
       this.queries.push(element.payload.namedQuery);
+      loadedNamedQuery = true;
```

Comment:
This task should take very little time. We should weigh the benefits of not raising a progress event for this amount of time over the confusion that might arise if we raise two progress events with the same values.

Reply:
Done.